tests/tests/smoke_tests.robot

```diff
@@ -49,20 +45,21 @@ Extract multiple files
     [Tags]    smoke
     [Template]    Extract test file
     FOR    ${file}    IN    @{TestFiles}
-        ${file}
+        ${TestFiles}[${file}]
     END
 
 Corrupted File Test
     [Documentation]    Test to check if file with corrupted header/non inno setup file shows error popup
     [Tags]    smoke    negative
+    ${test_file}    Set Variable    ${Corrupted_file}
```

Comment:
as above

Reply:
Agreed with Krzysztof that 'Set variable' topic will be handle by separate PR.